README.rst

```diff
@@ -23,22 +24,117 @@ implementation are in the accepted
 ``astropy_helpers`` includes a special "bootstrap" module called
 ``ah_bootstrap.py`` which is intended to be used by a project's setup.py in
 order to ensure that the ``astropy_helpers`` package is available for
-build/installation.  This is similar to the ``ez_setup.py`` module that is
-shipped with some projects to bootstrap `setuptools
-<https://bitbucket.org/pypa/setuptools>`_.
+build/installation.
 
-As described in APE4, the version numbers for ``astropy_helpers`` follow the
-corresponding major/minor version of the `astropy core package
-<http://www.astropy.org/>`_, but with an independent sequence of micro (bugfix)
-version numbers. Hence, the initial release is 0.4, in parallel with Astropy
-v0.4, which will be the first version  of Astropy to use ``astropy-helpers``.
+As described in `APE4 <https://github.com/astropy/astropy-APEs/blob/master/APE4.rst>`_, the version
+numbers for ``astropy_helpers`` follow the corresponding major/minor version of
+the `astropy core package <http://www.astropy.org/>`_, but with an independent
+sequence of micro (bugfix) version numbers. Hence, the initial release is 0.4,
+in parallel with Astropy v0.4, which will be the first version  of Astropy to
+use ``astropy-helpers``.
 
 For examples of how to implement ``astropy-helpers`` in a project,
 see the ``setup.py`` and ``setup.cfg`` files of the
 `Affiliated package template <https://github.com/astropy/package-template>`_.
 
+What does astropy-helpers provide?
+----------------------------------
+
+Custom setup.py commands
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The main part of astropy-helpers is to provide customized setuptools commands.
+For example, in a package that uses astropy-helpers, the following command
+will be available::
+
+    python setup.py build_docs
+
+and this command is implemented in astropy-helpers.
+
+The commands we provide or customize are:
+
+**python setup.py test**
+
+This command will automatically build the package, install it to a temporary
+directory, and run the tests using `pytest <http://pytest.org/>`_ on this
+installed version. Note that the bulk of this command is actually defined
+in ``astropy.tests.command.AstropyTest`` - here we simply define the custom
+setuptools command.
+
+**python setup.py sdist**
+
+We redefine ``sdist`` to use the version from distutils rather than from
+setuptools, as the setuptools version requires duplication of information
+in ``MANIFEST.in``.
```

Comment:
Is this customisation of `sdist` something that could maybe be removed completely?
I find `MANIFEST.in` confusing and don't understand what the good practices are concerning what include and exclude lines to put, i.e. whether to white or blacklist files, from the PYthon documentation.

So I can see why a modifcation was introduced here as an attempt to make it better.

But probably making it behave differently in Python packages that use astropy-helpers and "normal" Python packages that use setuptools isn't a good idea? It just creates more confusion overall?

Reply:
see #419 for the discussion or removal of the custom `sdist`.